.nvmrc

```diff
@@ -1,2 +1 @@
-4
-
+6.9.1
```

Comment:
6.9 is the LTS version of node. There are actually only two versions node of 6.9: 6.9.0 and 6.9.1 The former has issues that need to be fixed in 6.9.1. I don't see any reason why we should let people use versions below 6.9.